kb/how-to-explain-temporal.md

```diff
@@ -0,0 +1,188 @@
+---
+slug: how-to-explain-temporal
+title: How to explain Temporal
+tags:
+  - resources
+date: 2022-12-12T00:00:00Z
+---
+
+There are many ways to introduce and teach Temporal based on your background.
+Temporal doesn't have a monopoly on explaining Temporal.
+
+<!-- truncate -->
+
+Here, we list great blogposts and videos by Temporal staff and users that might be helpful in increasing your understanding or convincing your team.
+
+If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/kb/additional-resources.md) so that others can find it!
```

Comment:
```suggestion
If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/kb/how-to-explain-temporal.md) so that others can find it!
```